CHANGELOG.asciidoc

```diff
@@ -119,6 +119,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add experimental socket summary metricset to system module {pull}6782[6782]
 - Increase ignore_above for system.process.cmdline to 2048. {pull}8101[8100]
 - Add support to renamed fields planned for redis 5.0. {pull}8167[8167]
+- Added 'died' PID state to process_system metricset on system module
```

Comment:
Add also a reference to the pull request as in other entries.

Reply:
And you will also need to update the branch with master, there is a conflict in the changelog.

Reply:
Fixed both in 503c282cb245dd8cdc9a10b5aca862bc852c6abc